### ISSUE TITLE
A detail in the paper

### ISSUE THREAD

**Issue by Chuan1937** [chenyk1990/eqpolarity#1]
The paper is :"To enhance the performance of the proposed algorithm, we use a transfer learning strategy. We use 10% of the Texas dataset to fine-tune the pretrained model. Among them, 9% of the Texas data are used for training and 1% of the Texas data are used for validation. We use the optimal weights of the pretrained model, and fine-tune using 9% of the Texas dataset with 1% of the Texas dataset for validation to monitor the validation loss."

| unknown	 |   288897   |
| :---------------- | ---- |
| U	        |   14671   |
| D	        |   8663    |

The number of "unkonwn,U,D" is extremely unbalance. So how is data balancing done in txed?

**Comment by Chuan1937** [comment]
@chenyk1990 @omarmohamed15
I look forward to your reply, and I will cite your paper in the future.

**Comment by chenyk1990** [comment]
Thanks for the comment. Most waveforms do not include a polarity
measurement because polarity is not a routine step in daily analysis at
TexNet.

For transfer learning in Texas, we only use the available polarity
measurements from TXED. Let me know if you have any other questions.

Yangkang
